cmd/vpn_server/app/app.go

```diff
@@ -67,6 +70,40 @@ func run(_ context.Context, log logr.Logger) error {
 		return err
 	}
 
+	if !cfg.IsHA {
+		log.Info("setting up iptables rules for Envoy")
+		ipTable, err := network.NewIPTables(log, iptables.ProtocolIPv4)
+		if err != nil {
+			return err
+		}
+
+		for _, nw := range cfg.PodNetworks {
+			if nw.IsIPv4() {
+				err = ipTable.AppendUnique("nat", "OUTPUT", "-m", "owner", "--uid-owner", "65532", "-d", nw.String(), "-j", "NETMAP", "--to", constants.ShootPodNetworkMapped)
```

Comment:
Can we please pass the user id as environment variable instead of hard-coding the value? Otherwise, we have this value in two places and changing one will break things.

Reply:
I've added a constant to g/g in the [other PR](https://github.com/gardener/gardener/pull/11582) the idea was to import that constant after the gardener bump. so effectively there would be only one place where this is defined.